### PR TITLE
update docs to include write example for HCS dataset

### PR DESCRIPTION
**Pull request by elyall**

### CODE REVIEW

File: docs/source/python.rst [hunk 1]
```diff
@@ -80,6 +80,55 @@ This image can be viewed in `napari` using the
...
+
+This sample code shows how to write a high-content screening dataset (i.e. culture plate with multiple wells) to a OME-NGFF file::
+
+    import numpy as np
+    import zarr
+
+    from ome_zarr.io import parse_url
+    from ome_zarr.writer import write_image, write_plate_metadata, write_well_metadata
+
+    path = "test_ngff_image.zarr"
```
Comment (will-moore) [inline docs/source/python.rst:94]: This should be `file_path` and `plate` makes more sense than `image` here.
e.g. `file_path = "test_ngff_plate.zarr"`

File: docs/source/python.rst [hunk 2]
```diff
@@ -80,6 +80,55 @@ This image can be viewed in `napari` using the
...
+    mean_val=10
+    num_wells = len(well_paths)
+    num_fields = len(field_paths)
+    size_xy = 128
+    size_z = 10
+    rng = np.random.default_rng(0)
+    data = rng.poisson(mean_val, size=(num_wells, num_fields, size_z, size_xy, size_xy)).astype(np.uint8)
+
+    # write the plate of images and corresponding metadata
+    store = parse_url(str(file_path), mode="w").store
```
Comment (will-moore) [inline docs/source/python.rst:110]: could remove the `str()` here.

File: docs/source/python.rst (hunk 1 above)
Comment (elyall) [reply docs/source/python.rst]: Thanks for the catch! I missed this when converting my code to match the style of the documentation. I instead changed `file_path` to `path` to match the other sections. Now fixed.

**Comment by will-moore** [comment]
Working great, thx.

**Comment by elyall** [comment]
@will-moore: Thanks for merging this, but I have noticed [the website](https://ome-zarr.readthedocs.io/en/stable/python.html) has not yet updated to reflect the changes. Please advise.

**Comment by jburel** [comment]
In order to update the "stable" branch on readthedocs, a tag will need to be pushed.
the merged changes are available on "latest" branch i.e. https://ome-zarr.readthedocs.io/en/latest/python.html

**Comment by elyall** [comment]
Ah whoops, thanks.

From the sidebar it looks like I got the RST heading wrong specifying a level 1 heading instead of a level 3. Any idea what underlining I should've use?

**Comment by jburel** [comment]
When a PR is open a dedicated build is done on readthedocs e.g. this PR https://ome-zarr--317.org.readthedocs.build/en/317/. It is rebuilt on every commit

So it is easy to review your changes and test option

**Comment by jburel** [comment]
Formatting adjusted in https://github.com/ome/ome-zarr-py/pull/320

**Pull request merged**